src/braket/experimental/autoqasm/program/program.py

```diff
@@ -470,7 +470,6 @@ def validate_gate_targets(self, qubits: list[Any], angles: list[Any]) -> None:
             angles (list[Any]): The list of target angles to validate.
 
         Raises:
-            errors.InvalidTargetQubit: Target qubits are invalid in the current context.
```

Comment:
Did you mean to remove this? It looks like the call to `_validate_verbatim_target_qubits` could still raise this error.

Reply:
Ah good call, I totally missed that lol